setup/install/providers/aws/aws-eks.md

```diff
@@ -0,0 +1,200 @@
+#kubectl apply -f https://raw.githubusercontent.com/spinnaker/spinnaker.github.io/master/setup/install/providers/aws/eks/eks-admin-cluster-role-binding.yaml 
+kubectl apply -f https://raw.githubusercontent.com/spinnaker/spinnaker.github.io/master/setup/install/providers/aws/eks/spinnaker-service-account.yaml 
+kubectl apply -f https://raw.githubusercontent.com/spinnaker/spinnaker.github.io/master/setup/install/providers/aws/eks/spinnaker-cluster-role-binding.yaml 
+TOKEN=$(kubectl get secret --context $CONTEXT \
+   $(kubectl get serviceaccount spinnaker-service-account \
+       --context $CONTEXT \
+       -n spinnaker \
+       -o jsonpath='{.secrets[0].name}') \
+   -n spinnaker \
+   -o jsonpath='{.data.token}' | base64 --decode)
+kubectl config set-credentials ${CONTEXT}-token-user --token $TOKEN
+kubectl config set-context $CONTEXT --user ${CONTEXT}-token-user
+```
+
+> This point onwards, the location (either EC2 instance, or container or local) where you run halyard commands from should have the kubeconfig file that was created
+by the above kubectl commands. If you ran the kubectl commands locally and want to run halyard command on an instance, then you can copy the kubeconfig file
+from local to the instance.
+
+## Enable Kubernetes Cloud provider using Halyard
+
+```
+hal config provider kubernetes enable
+hal config provider kubernetes account add ${MY_K8_ACCOUNT}   --provider-version v2   --context $(kubectl config current-context)
+hal config features edit --artifacts true
+```
+
+## Create Kubernetes worker nodes
+
+Worker nodes use [EKS optimized AMIs](https://docs.aws.amazon.com/eks/latest/userguide/worker.html)
+```
+curl -O https://raw.githubusercontent.com/spinnaker/spinnaker.github.io/master/setup/install/providers/aws/eks/amazon-eks-nodegroup.yaml  
+aws cloudformation deploy --stack-name spinnaker-eks-nodes --template-file amazon-eks-nodegroup.yaml \
+--parameter-overrides NodeInstanceProfile=$SPINNAKER_INSTANCE_PROFILE_ARN \
+NodeInstanceType=t2.large ClusterName=$EKS_CLUSTER_NAME NodeGroupName=spinnaker-cluster-nodes ClusterControlPlaneSecurityGroup=$CONTROL_PLANE_SG \
+Subnets=$SUBNETS VpcId=$VPC_ID --capabilities CAPABILITY_NAMED_IAM
+
+```
+
+## Join the nodes created above with Spinnaker EKS cluster and watch them to appear in ready state
+
+Replace `<spinnaker-role-arn>` with $AUTH_ARN and save it as aws-auth-cm.yaml
+
+```
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: aws-auth
+  namespace: kube-system
+data:
+  mapRoles: |
+    - rolearn: <spinnaker-role-arn>
+      username: system:node:{{EC2PrivateDNSName}}
+      groups:
+        - system:bootstrappers
+        - system:nodes
+
+```
+
+Run following to join nodes with the cluster
+
+```
+kubectl apply -f aws-auth-cm.yaml
+```
+
+Watch the nodes to come in ready state
+
+```
+kubectl get nodes --watch
+```
+
+
+## Next steps
+
+Optionally, you can [set up Amazon's Elastic Container
```

Comment:
To be consistent with the other pages, please phrase this like seen on other provider installation pages.